active/NumberBox/NumberBox.md

```diff
@@ -159,24 +185,31 @@ unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.Control
     String Text; 
     String PlaceholderText;
 
+<!-- todo: Why does this say "basic"? Is there also an advanced validation mode? -->   
     NumberBoxBasicValidationMode BasicValidationMode;
 
     Boolean AcceptsCalculation;
 
     NumberBoxSpinButtonPlacementMode SpinButtonPlacementMode{ get; set; };
 
+<!-- todo: This should have the "Is" prefix, to differentiate them from events. -->
     Boolean HyperScrollEnabled;
     Boolean HyperDragEnabled;
     Boolean WrapEnabled;
 
+<!-- NumberFormatter is worth an example -->
     Windows.Globalization.NumberFormatting.INumberFormatter2 NumberFormatter;
 
+<!-- todo: Suggest DecrementButtonIcon naming, to match DecrementButtonStyle -->
+
     IconSource DecrementIcon;
     IconSource IncrementIcon;
     Windows.UI.Xaml.Style DecrementButtonStyle;
     Windows.UI.Xaml.Style IncrementButtonStyle;
     NumberBoxTemplateSettings TemplateSettings{ get; };
 
+<!-- todo: Do we need ValueChanging? Assume we do, the Remarks or API Notes section should explain it's behavior. Does this respond to TextBox's TextChanging or BeforeTextChanging events? -->
```

Comment:
I also would like to know that -- we don't currently have it. But no, since we don't evaluate when the textbox text changes we don't listen to or handle TextChanging.

Reply:
Actually, instead of "respond" I meant to say "correspond". 

On the one hand, I'd expect this event to correspond to [TextBox.TextChanging](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.TextBox.TextChanging), based simply on the name. If so, we should call that out. On the other hand, TextBox.TextChanging has some complicated semantics, so it's reasonable to consider breaking from that. But that should be called out as well.

Reply:
ValueChanging was requested before we planned built-in input validation for NumberBox. I am finding its value limited now. Correct me if I am wrong, but I believe @mrlacey originally called out the absence of ValueChanging. @mrlacey - I know your time is valuable, so I hope this is a quick question, but are there any scenarios in which you are looking to use NumberBox that would be disabled if ValueChanging were absent? 

Otherwise, I also believe it is acceptable to break from TextBox event parity considering the addition and ease of our coming input validation for NumberBox.

Reply:
Also, what is the purpose of the args `IsContentChanging` property? It's not obvious from the name, and not explained anywhere.

Reply:
This was an artifact of originally intending event parity with TextBox's ValueChanged and ValueChanging events and their [args](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.textboxtextchangingeventargs). 

If we're in agreement to cut ValueChanging from NumberBox, IsContentChanging would be dropped as well.